src/utils/validationFunctions.js

```diff
@@ -72,3 +72,21 @@ export function validateCanModify(targetUserId, user) {
   }
   return true
 }
+
+export function validateCreateCohort(name, users, startDate, endDate) {
+  if (!name) {
```

Comment:
can you accept an empty name? ie name === ""?

Reply:
No this should not be accepted. When I run it now with a name of "", we get an error back that name is required!